src/features/click-to-load.js

```diff
@@ -2011,19 +2011,13 @@ export default class ClickToLoad extends ContentFeature {
     get messaging () {
         if (this._messaging) return this._messaging
 
-        if (this.platform.name === 'android' || this.platform.name === 'extension' || this.platform.name === 'macos') {
+        if (this.platform.name === 'extension' || this.platform.name === 'macos') {
             this._clickToLoadMessagingTransport = new SendMessageMessagingTransport()
             const config = new TestTransportConfig(this._clickToLoadMessagingTransport)
             this._messaging = new Messaging(this.messagingContext, config)
             return this._messaging
-        } else if (this.platform.name === 'ios') {
-            const config = new WebkitMessagingConfig({
```

Comment:
It looks like we don't need anything special for iOS do we? If so we can use the super implementation also.